src/Uno.UWP/Graphics/Imaging/SoftwareBitmap.skia.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using SkiaSharp;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Runtime.InteropServices;
+
+namespace Windows.Graphics.Imaging
+{
+	static class SoftwareBitmapExtensions
+	{
+		[global::System.Runtime.CompilerServices.MethodImpl(global::System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
+		public static SKColorType ToSKColorType(this BitmapPixelFormat format) =>
+			format switch
+			{
+				BitmapPixelFormat.Unknown => SKColorType.Unknown,
+				BitmapPixelFormat.Rgba16 => SKColorType.Rgba16161616,
+				BitmapPixelFormat.Rgba8 => SKColorType.Rgba8888,
+				BitmapPixelFormat.Gray8 => SKColorType.Gray8,
+				BitmapPixelFormat.Bgra8 => SKColorType.Bgra8888,
+				_ => throw new global::System.NotSupportedException(nameof(format))
+			};
+
+		[global::System.Runtime.CompilerServices.MethodImpl(global::System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
+		public static BitmapPixelFormat ToBitmapPixelFormat(this SKColorType format) =>
+			format switch
+			{
+				SKColorType.Unknown => BitmapPixelFormat.Unknown,
+				SKColorType.Rgba16161616 => BitmapPixelFormat.Rgba16,
+				SKColorType.Rgba8888 => BitmapPixelFormat.Rgba8,
+				SKColorType.Gray8 => BitmapPixelFormat.Gray8,
+				SKColorType.Bgra8888 => BitmapPixelFormat.Bgra8,
+				_ => throw new global::System.NotSupportedException(nameof(format))
+			};
+
+		[global::System.Runtime.CompilerServices.MethodImpl(global::System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
+		public static SKAlphaType ToSKAlphaType(this global::Windows.Graphics.Imaging.BitmapAlphaMode alpha) =>
+			alpha switch
+			{
+				BitmapAlphaMode.Ignore => SKAlphaType.Opaque,
+				BitmapAlphaMode.Straight => SKAlphaType.Unpremul,
+				BitmapAlphaMode.Premultiplied => SKAlphaType.Premul,
+				_ => throw new global::System.NotSupportedException(nameof(alpha))
+			};
+
+		[global::System.Runtime.CompilerServices.MethodImpl(global::System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
+		public static BitmapAlphaMode ToBitmapAlphaMode(this SKAlphaType alpha) =>
+			alpha switch
+			{
+				SKAlphaType.Opaque => BitmapAlphaMode.Ignore,
+				SKAlphaType.Unpremul => BitmapAlphaMode.Straight,
+				SKAlphaType.Premul => BitmapAlphaMode.Premultiplied,
+				_ => throw new global::System.NotSupportedException(nameof(alpha))
+			};
+	}
+
+	public partial class SoftwareBitmap
```

Comment:
@jeromelaban 
Do I have to implement a specific backend for each platform or can I use Skia for all of them?

Reply:
As I thought. I have no way to test on the Apple platform.